packages/types/templates.ts

```diff
@@ -1,16 +1,31 @@
 import { z } from "zod";
-
 import { ZLegacySurveyQuestions, ZLegacySurveyThankYouCard, ZLegacySurveyWelcomeCard } from "./LegacySurvey";
 import { ZSurveyHiddenFields, ZSurveyQuestions, ZSurveyThankYouCard, ZSurveyWelcomeCard } from "./surveys";
 import { ZUserObjective } from "./user";
 
+export const ZTemplateChannel = z.enum(["Website Survey", "App Survey", "Email Survey", "Link Survey"]);
+export type TTemplateChannel = z.infer<typeof ZTemplateChannel>;
+
+export const ZTemplateIndustry = z.enum([
```

Comment:
Also here I would use camel-case config variables / enums.
`saas`, `eCommerce`, `other`.

Since the onboarding we are also working an for now only supports these 3 I would also here limit them to those 3 for now and add new ones if required later.